src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -55,6 +59,14 @@ public class MeasureController {
   private final GroupService groupService;
   private final ActionLogService actionLogService;
 
+  @GetMapping("/measures/draftstatus")
+  public ResponseEntity<Map<String, Boolean>> getDraftStatuses(
+      @RequestParam(required = false, name = "measureSetIds") List<String> measureSetIds) {
```

Comment:
I don't see an obvious null check on `measureSetIds`, should the param be required?

Reply:
yes